tests/dragonfly/cluster_test.py

```diff
@@ -142,3 +148,121 @@ async def test_cluster_nodes(async_client):
     assert info['last_ping_sent'] == '0'
     assert info['slots'] == [['0', '16383']]
     assert info['master_id'] == "-"
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_cluster(df_local_factory):
```

Comment:
lets be more specific in the test name. What are we testing in this test case?
Also adding a comment above the test for the steps in the test is good.